gdx/src/com/badlogic/gdx.gwt.xml

```diff
@@ -417,7 +417,6 @@
 		<include name="utils/QuickSelect.java"/>
 		<include name="utils/ReflectionPool.java"/>
 		<include name="utils/Scaling.java"/>
-		<exclude name="utils/ScreenUtils.java"/> <!-- Reason: Type mismatch Buffer->ByteBuffer -->
```

Comment:
Is it your intention to include this? If so, you should to change from exclude to include, rather than just removing.